msal/src/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs

```diff
@@ -145,4 +146,26 @@ public partial class TokenCacheNotificationArgs
         [Obsolete("Use Account instead (See https://aka.ms/msal-net-2-released)", true)]
         public IUser User { get { throw new NotImplementedException(); } }
     }
+
+    /// <Summary>
+    /// Abstract class containing common API methods and properties. 
+    /// For details see https://aka.ms/msal-net-client-applications
+    /// </Summary>
+    public partial class PublicClientApplication
```

Comment:
>PublicClientApplication [](start = 25, length = 23)

We never had this method in MSAL, why do we want to obselete an inexistent method?

Reply:
This is to help those upgrading from ADAL trying to use the string version of the pwd. Telling them to use the SecureString intead.